packages/xlsx-renderer-cli/src/program.ts

```diff
@@ -0,0 +1,70 @@
+import { Command, default as commander } from 'commander';
+import * as fs from 'fs';
+import { Stream } from 'stream';
+import { Renderer } from 'xlsx-renderer';
+const version = '0.0.1';
```

Comment:
It could be read directly from package.json

Reply:
done in 5b77b06